README.md

```diff
@@ -0,0 +1,42 @@
+<h4> 1. The purpose of clients-libs-java-samples </h4>
```

Comment:
Lets also add initial section `What is this` with info such as: `this is a super simple CLI based sample showing how one could use Adscore API within his code.`

Reply:
Added!

Reply:
Please also change MR desc :)